scripts/upload.gradle

```diff
@@ -128,24 +130,20 @@ afterEvaluate { project ->
         }
 
         task androidJavadocsJar(type: Jar, dependsOn: androidJavadocs) {
+            classifier "javadoc"
             from androidJavadocs.destinationDir
             baseName "${JAR_PREFIX}${project.name}${JAR_POSTFIX}"
         }
 
         task androidSourcesJar(type: Jar) {
+            classifier = 'sources'
```

Comment:
He means the quotes difference, in case you didn't see; it took me a few seconds to notice.